nautobot/core/testing/filters.py

```diff
@@ -286,6 +287,16 @@ def test_q_filter_valid(self):
                         obj, obj_field_name = self._get_nested_related_obj_and_its_field_name(obj, obj_field_name)
                     self._assert_q_filter_predicate_validity(obj, obj_field_name, filter_field_name, lookup_method)
 
+        def test_content_type_related_fields_uses_content_type_filter(self):
+            for field in self.queryset.model._meta.fields:
+                related_model = getattr(field, "related_model", None)
+                if not related_model or related_model != ContentType:
+                    continue
+                with self.subTest(
+                    f"Assert {self.filterset.__class__.__name__}.{field.name} implements ContentTypeFilter"
+                ):
+                    self.assertIsInstance(self.filterset.get_filters().get(field.name), ContentTypeFilter)
```

Comment:
Failing in CI because any CableTermination models have a `_cable_peer_type` model field that doesn't have a same-named filter field.